Assets/MixedRealityToolkit/_Core/Inspectors/ControllerPopupWindow.cs

```diff
@@ -0,0 +1,723 @@
+                            ids = singleAxisActionIds;
+                            break;
+                        case AxisType.DualAxis:
+                            labels = dualAxisActionLabels;
+                            ids = dualAxisActionIds;
+                            break;
+                        case AxisType.ThreeDofPosition:
+                            labels = threeDofPositionActionLabels;
+                            ids = threeDofPositionActionIds;
+                            break;
+                        case AxisType.ThreeDofRotation:
+                            labels = threeDofRotationActionLabels;
+                            ids = threeDofRotationActionIds;
+                            break;
+                        case AxisType.SixDof:
+                            labels = sixDofActionLabels;
+                            ids = sixDofActionIds;
+                            break;
+                    }
+
+                    EditorGUI.BeginChangeCheck();
+                    actionId.intValue = EditorGUILayout.IntPopup(GUIContent.none, actionId.intValue, labels, ids, GUILayout.Width(InputActionLabelWidth));
+
+                    if (EditorGUI.EndChangeCheck())
+                    {
+                        var inputAction = actionId.intValue == 0 ? MixedRealityInputAction.None : MixedRealityManager.Instance.ActiveProfile.InputActionsProfile.InputActions[actionId.intValue - 1];
+                        actionDescription.stringValue = inputAction.Description;
+                        actionConstraint.enumValueIndex = (int)inputAction.AxisConstraint;
+                    }
+
+                    if ((AxisType)axisType.intValue == AxisType.Digital)
+                    {
+                        var keyCode = interaction.FindPropertyRelative("keyCode");
+                        EditorGUILayout.PropertyField(keyCode, GUIContent.none, GUILayout.Width(InputActionLabelWidth));
+                    }
+                    else
+                    {
+                        if ((AxisType)axisType.intValue == AxisType.DualAxis)
+                        {
+                            EditorGUIUtility.labelWidth = InputActionLabelWidth * 0.5f;
+                            EditorGUIUtility.fieldWidth = InputActionLabelWidth * 0.5f;
+
+                            int currentAxisSetting = 0;
+
+                            if (invertXAxis.boolValue)
+                            {
+                                currentAxisSetting += 1;
+                            }
+
+                            if (invertYAxis.boolValue)
+                            {
+                                currentAxisSetting += 2;
+                            }
+
+                            EditorGUI.BeginChangeCheck();
+                            currentAxisSetting = EditorGUILayout.IntPopup(InvertContent, currentAxisSetting, InvertAxisContent, InvertAxisValues, GUILayout.Width(InputActionLabelWidth));
```

Comment:
Doesn't look like #2620 was integrated/rewritten correctly. Also not sure why the change in the list was made, causing the display to go from "Invert Y" to "Invert Invert Y".

Before: 
![image](https://user-images.githubusercontent.com/3580640/44633016-c8afe200-a939-11e8-93d6-d505d0b57628.png)

After:
![image](https://user-images.githubusercontent.com/3580640/44633018-d06f8680-a939-11e8-82ce-408545aa0400.png)

Reply:
I was integrated correctly, and actually fixed a few issues. 

I'll remove the duplicate invert.

Reply:
Where did the "Invert X Axis?" fields go?

Reply:
Oh I see. On tick, I'll be sure to update it.

Reply:
Great catch. Fixed

Reply:
>and actually fixed a few issues.

For future reference, what specific issues did you fix, and are they things we need to keep an eye out for?

Reply:
Take a look at the description of the issue where I laid them out plainly.

Reply:
Which issue? The only issue mentioned in this PR is the one about adding the pop-up windows.

Are you referring to the constructor updates mentioned in this PR?

Reply:
> Fixed an issue with constructors not being updated after the X Axis Inversion addition.

Yes